Traceback (most recent call last):
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.10/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 641, in run_until_complete
    return future.result()
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.10/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.10/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.10/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from skimage.feature import ORB

# Initialize ORB
# This number of keypoints is large enough for robust results, 
# but low enough to run within a few seconds. 
orb = ORB(n_keypoints=800, fast_threshold=0.05)

# Detect keypoints in pano0
orb.detect_and_extract(pano0)
keypoints0 = orb.keypoints
descriptors0 = orb.descriptors

# Detect keypoints in pano1 and pano2
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Input [0;32mIn [7][0m, in [0;36m<module>[0;34m[0m
[1;32m      6[0m orb [38;5;241m=[39m ORB(n_keypoints[38;5;241m=[39m[38;5;241m800[39m, fast_threshold[38;5;241m=[39m[38;5;241m0.05[39m)
[1;32m      8[0m [38;5;66;03m# Detect keypoints in pano0[39;00m
[0;32m----> 9[0m orb[38;5;241m.[39mdetect_and_extract([43mpano0[49m)
[1;32m     10[0m keypoints0 [38;5;241m=[39m orb[38;5;241m.[39mkeypoints
[1;32m     11[0m descriptors0 [38;5;241m=[39m orb[38;5;241m.[39mdescriptors

[0;31mNameError[0m: name 'pano0' is not defined
NameError: name 'pano0' is not defined

